README.md

```diff
@@ -237,6 +237,55 @@ spec:
 
 Create a PVC using the storage class created for the ZFS driver.
 
+1. If storageclass has fstype as ext4/3/2, xfs, btrfs then we have to create the PVC with storage size which is multiple of volblocksize mentioned in the storageclass. The same is also true if fstype is not mentioned in the storageclass in which case dafault will be ext4.
+
+```
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: openebs-zfspv
+parameters:
+  volblocksize: "4k"
+  fstype: "ext4"
+  poolname: "zfspv-pool"
+provisioner: zfs.csi.openebs.io
+```
+
+Here, PVC should have the storage multiple of volblocksize which is 4k (4 * 1024). If is not the multiple of volblocksize then ZFS will not create the volume and pod will not be able to come into RUNNING state as mount will keep on failing. For example, if we have created the below PVC with the above storageclass
```

Comment:
@pawanpraka1 - can we someone avoid this manual calculation. When setting the value on the zfs, can we not round it up to the next highest number which  is a multiple of the block/record size?

Reply:
that will violate the concept of pvc claim. Provisioned size will be more than claimed size. Provisioned size should be equal to PVC size. Doing round up would not be the right approach IMHO.

Reply:
Hmm.. so it is about usability vs bytes. another case where this is accepted in practice is when we format a filesystem of 100G, it doesn't always give us 100G usable capacity, so a few bytes is ok in that perspective. 

It is probably harder to do the math that the size should be strictly a multiple of 1024. It has to be rounded off to the closest multiple, I am assuming the diff will be a max of record size == like 4k to 128k bytes difference. 

It is very common for users to request for size as: 
- 5G
- 5Gi

If either of them fails, it violates the ease of use principle.

Reply:
@kmova I somehow don't feel that is the right thing. are we doing the same thing in cstor? How are we handeling it there?

Reply:
as per discussion rounding off the size seems to be good.